go/pkg/pass1/types.go

```diff
@@ -478,6 +487,7 @@ type zone struct {
 	primaryMark          int
 	secondaryMark        int
 	statefulMark         int
+	toZone1              *routerIntf
```

Comment:
Wozu brauchen wir an der Zone eigentlich das Attribut toZone1, denn pathWalk kam ja offenbar ohne aus.
Es wird also nur lokal in setPath verwendet. Müssen wir dafür ein Attribut hinzufügen, das für immer an jeder Zone hängt?

Reply:
das war tatsächlich ein Überbleibsel vom Anfang, als ich das pathObj - Interface noch nicht benutzt habe und wurde überhaupt nicht gebraucht.